lib/rules/__tests__/index.test.mjs

```diff
@@ -32,16 +32,17 @@ describe('all rules', () => {
 	});
 });
 
-describe('fixable rules', () => {
-	test.each(ruleNames)('"%s" should describe fixable in the documents', async (name) => {
+describe('fixability', () => {
+	test.each(ruleNames)('"%s" should be properly documented', async (name) => {
 		const rule = await rules[name];
-
-		if (!rule.meta.fixable) return;
-
+		const cell = rule.meta.fixable ? '\\s+🔧\\s+' : ' ';
+		const rulesPattern = `^.+\`${name}\`.+\\|.+\\|${cell}\\|$`;
+		const needle = '`fix` option';
+		const rulePattern = rule.meta.fixable ? needle : `^((?!${needle})(\\s|\\S))*$`;
```

Comment:
[suggest] I feel a bit more readable if the definitions are in the same order as `expect(...)` lines. How about this?

- `rulePatter` -> `expect(ruleDoc)`
- `rulesPattern` -> `expect(rulesListDoc)`

```suggestion
		const needle = '`fix` option';
		const rulePattern = rule.meta.fixable ? needle : `^((?!${needle})(\\s|\\S))*$`;
		const rulesPattern = `^.+\`${name}\`.+\\|.+\\|${cell}\\|$`;
```